packages/standard-asset-apis/test/RoutedSender-spec.ts

```diff
@@ -1,33 +1,49 @@
 /* eslint-disable @typescript-eslint/naming-convention */
-import { DataEntity, pDelay, RouteSenderAPI } from '@terascope/utils';
+import {
+    DataEntity,
+    pDelay,
+    RouteSenderAPI,
+    Logger
+} from '@terascope/utils';
 import 'jest-extended';
 import {
     RoutedSender
 } from '../src';
 
+const logger: any = {
```

Comment:
so this can break if log levels are set to trace or warn, you should just use the debugLogger from utils, as that is a proper logger so you don't have to force that. It how we loggers for almost all of our tests.